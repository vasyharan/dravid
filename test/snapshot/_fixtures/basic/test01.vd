def foo(a, b, c) {
   val bar = (a + 10) * b - 1 / c
   bar
}

def main() {
  foo(1, 2, 3)
}