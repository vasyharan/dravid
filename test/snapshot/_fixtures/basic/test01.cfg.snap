(bb 
   (val d
          (*
           (int 10)
           (id c)))
   (id d))
(bb 
   (+
     (id a)
     (int 10)))
(bb 
   (id a))
(bb 
   (-
     (id a)
     (int 10)))
(bb 
   (*
     (id a)
     (int 10)))
(bb 
   (call foo
           (int 1)
           (int 2)
           (int 3))
   (call foo1
           (int 1)
           (int 2)
           (int 3))
   (call foo2
           (int 1)
           (int 2)
           (int 3)))
(bb 
   (+
     (+
      (int 1)
      (int 2))
     (id x)))
(bb 
   (*
     (+
      (+
       (int 1)
       (int 2))
      (id x))
     (+
      (id x)
      (+
       (int 1)
       (int 2)))))
